jina/serve/runtimes/request_handlers/data_request_handler.py

```diff
@@ -169,6 +190,7 @@ async def handle(self, requests: List['DataRequest']) -> DataRequest:
                 requests,
                 field='docs',
             ),
+            otel_context=otel_context,
```

Comment:
we have to rethink the name @JohannesMessner @samsja @alaeddine-13 any suggestion?

Reply:
`tracing_context` ?
we use `monitoring` as reference to prometheus, we can come up with a term like `tracing` to reference open telemetry as well.